test/run/ch-run_misc.bats

```diff
@@ -367,7 +367,7 @@ chse_dockerfile=foo
 EOF
 )
 
-    run ch-run --set-env="${img}/environment" "$img" -- \
+    run ch-run --set-env="${img}/ch/environment" "$img" -- \
```

Comment:
We need to update the ch-run command reference, section `4.7.5.2. Setting variables with --set-env` to reflect this change.

Reply:
Done.